sale_invoice_plan/views/sale_view.xml

```diff
@@ -0,0 +1,183 @@
+                        <field name="to_invoice" />
+                    </group>
+                    <group>
+                        <field name="invoice_type" />
+                        <field name="percent" />
+                        <field name="invoiced" />
+                        <field name="invoice_move_ids" widget="many2many_tags" />
+                    </group>
+                </group>
+                <separator
+                    string="Related Invoices"
+                    attrs="{'invisible': [('invoice_move_ids', '=', [])]}"
+                />
+                <field name="invoice_move_ids" widget="many2many_tags" />
+            </form>
+        </field>
+    </record>
+    <record id="view_order_form_invoice_plan" model="ir.ui.view">
+        <field name="name">view.order.form.invoice.plan</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr='/form/sheet/notebook' position="before">
+                <div attrs="{'readonly': [('invoice_count', '>', 0)]}">
+                    <field name="use_invoice_plan" />
+                    <label for="use_invoice_plan" />
+                </div>
+            </xpath>
+            <xpath expr="//button[@name='payment_action_capture']" position="before">
+                <field name="ip_invoice_plan" invisible="1" />
+                <button
+                    name="%(action_view_sale_make_planned_invoice)d"
+                    string="Create Invoice by Plan"
+                    type="action"
+                    class="btn-primary"
+                    attrs="{'invisible': [('ip_invoice_plan', '=', False)]}"
+                />
+            </xpath>
+            <xpath expr="/form/sheet/notebook/page" position="after">
+                <page
+                    string="Invoice Plan"
+                    attrs="{'invisible': [('use_invoice_plan', '=', False)]}"
+                >
+                    <button
+                        name="%(action_sale_create_invoice_plan)d"
+                        string="⇒ Create Invoice Plan"
+                        type="action"
+                        class="oe_link"
+                        attrs="{'invisible': ['|', ('invoice_plan_ids', '!=', []), ('invoice_count', '>', 0)]}"
+                    />
+                    <button
+                        name="remove_invoice_plan"
+                        string="⇒ Remove Invoice Plan"
+                        type="object"
+                        class="oe_link"
+                        attrs="{'invisible': ['|', ('invoice_plan_ids', '=', []), ('invoice_count', '>', 0)]}"
+                        confirm="Are you sure to remove this invoice plan?"
+                    />
+                    <field
+                        name="invoice_plan_ids"
+                        context="{'tree_view_ref': 'view_sale_invoice_plan_tree'}"
```

Comment:
@kittiu Could you make a fix for this. This generates a warning on view load. 

See https://travis-ci.com/github/OCA/sale-workflow/jobs/494482747#L1762

Thanks !